index.js

```diff
@@ -728,6 +727,75 @@ module.exports = {
     })
   },
 
+  /**
+   * Gets a statistic on one field at a time
+   * Supports where and geometry filters and group by
+   * @param {String} table to get data from
+   * @param {String} field to generate stats from
+   * @param {String} outName the name of the stat field
+   * @param {String} type - the stat type: min, max, avg, count
```

Comment:
also, no var or std dev? IMO we should support everything that is currently supported in lib/featureservice